src/Components/test/testassets/ComponentsApp.App/Pages/Ticker.cshtml

```diff
@@ -0,0 +1,21 @@
+@page "/ticker"
+
+<h1>Ticker</h1>
+<p class="tick-value">@tick</p>
+
+@functions {
+    int tick;
+
+    protected override void OnInit()
+    {
+        Task.Run(() => InvokeAsync(async () =>
+        {
+            while (true)
```

Comment:
This will continue forever, even after the user has navigated away, won't it? Probably not an example we want people to see any copy 😄 

I know it's more verbose, but can you do something to make it stop when the component itself is disposed (e.g., a cancellation token)?